web/src/hooks/layers.js

```diff
@@ -4,14 +4,20 @@ import { interpolate } from 'd3-interpolate';
 import { formatDistance } from 'date-fns';
 
 import { getRefTime, getTargetTime } from '../helpers/grib';
+import { TIME } from '../helpers/constants';
 
 export function useExchangeArrowsData() {
   const isConsumption = useSelector((state) => state.application.electricityMixMode === 'consumption');
+  const isHourly = useSelector((state) => state.application.selectedTimeAggregate === TIME.HOURLY);
```

Comment:
Wondering if it's matters if the equality check happens inside the useSelector or outside (like below)? 🤔 
Just thinking it might be that the variables can't be trusted in useMemo below due to this, but I have no clue...
If not, then just ignore this!
```suggestion
  const isHourly = useSelector((state) => state.application.selectedTimeAggregate) === TIME.HOURLY;
```